### ISSUE TITLE
Feature : Custom arg serialization per endpoint

### ISSUE THREAD

**Issue by michal-kurz** [reduxjs/redux-toolkit#2491]
I want to be able to provide custom arg serializer per endpoint. @phryneas [told me on stackoverflow](https://stackoverflow.com/a/72802392/10467064) that a PR with minimal overhead would be welcome. I have a solution, but cannot push into the repo to make a PR:

```
ERROR: Permission to reduxjs/redux-toolkit.git denied to michal-kurz.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
```

Can I please get the needed permissions?
Thank you 🙏

**Comment by phryneas** [comment]
You need to fork the repo, push it into a branch on your own repo and then open a pull request.

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

**Comment by michal-kurz** [comment]
@phryneas Alright, I've done that! 

PR is here: https://github.com/reduxjs/redux-toolkit/pull/2493
with some additional questions/info in description

**Comment by markerikson** [comment]
Just merged #2493 , so I'm going to close this. Thanks!
